app/Http/Middleware/ReactAdminResponse.php

```diff
@@ -26,8 +26,12 @@ public function handle(Request $request, Closure $next): Response
         $response = $next($request);
         if($request->routeIs('*.index')) {
             abort_unless(property_exists($request->route()->controller, 'modelclass'), 500, "It must exists a modelclass property in the controller.");
-            $modelClassName = $request->route()->controller->modelclass;
-            $response->header('X-Total-Count',$modelClassName::count());
+            $countRegister = $request->countRegister;
+            if($countRegister) {
```

Comment:
Bien este if, pero teneis el   abort_unless(property_exists($request->route()->controller, 'modelclass'), 500, "It must exists a modelclass property in the controller."); para controlar la variable vacio, pero desde mi punto de vista a si vale.